manifests/init.pp

```diff
@@ -168,7 +170,7 @@
   Optional[Hash]    $http_ports                    = $squid::params::http_ports,
   Optional[Hash]    $https_ports                   = $squid::params::https_ports,
   Optional[Hash]    $icp_access                    = $squid::params::icp_access,
-  Optional[String]  $logformat                     = $squid::params::logformat,
```

Comment:
If editing an .erb file would be good to update to it .epp  but up to you.

Reply:
Rebase done.  Did not do a rewrite, that's a level of scope change that I can't take on yet.